### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#514] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/

除了 VuePress 本身新增了一些 Markdown 语法外，vuepress-theme-hope 通过 ，在 Markdown 中启用了更多的语法与新功能。

https://theme-hope.vuejs.press/zh/guide/markdown/

**Comment by cupid007** [comment]
好

**Comment by lingdu990130556** [comment]
〇°
